composer.json

```diff
@@ -15,6 +15,7 @@
     ],
     "require": {
         "php": "^5.3.6 || >=7.0 <7.2",
+        "doctrine/annotations": "^1.2",
```

Comment:
Probably a bad idea to add this dependency since they've already dropped php 5.3->5.5 support, so I can only see this causing future dependency conflicts.